python_modules/dagster/dagster_tests/definitions_tests/test_partitioned_schedule.py

```diff
@@ -17,7 +17,7 @@
 
 def time_window(start: str, end: str) -> TimeWindow:
     return TimeWindow(
-        cast(datetime, pendulum.parser.parse(start)), cast(datetime, pendulum.parse(end))
+        cast(datetime, pendulum.parser.parse(start)), cast(datetime, pendulum.parser.parse(end))
```

Comment:
What happens if we don't do this? Most pendulum examples use `pendulum.parse`, so I'm skeptical of using `pendulum.parser.parse`. Could we turn off this rule?

Reply:
Ah yes this is a relic from an old commit, we have now turned off the rule `reportPrivateSymbolAccess` so I was able to revert this.